test/SlowTests/Tests/ResultsTransformer/NullDynamicValuesInTransformerOperation.cs

```diff
@@ -1,60 +0,0 @@
-using FastTests;
-using Raven.Client.Documents;
-using Raven.Client.Documents.Operations.Transformers;
-using Raven.Client.Documents.Transformers;
-using Sparrow.Json;
-using Xunit;
-
-namespace SlowTests.Tests.ResultsTransformer
-{
-    public class NullDynamicValuesInTransformerOperation : RavenTestBase
```

Comment:
Restore candidate:  here we are testing   + - / % operators of `DynamicNullObject` and check results return `null` then